doc/api/errors.md

```diff
@@ -563,6 +570,15 @@ found [here][online].
   encountered by [`http`][] or [`net`][] -- often a sign that a `socket.end()`
   was not properly called.
 
+<a id="nodejs-error-codes"></a>
+## Node.js Error Codes
+
+<a id="ERR_UNKNOWN_ENCODING"></a>
+### ERR_UNKNOWN_ENCODING
+
+The `'ERR_UNKNOWN_ENCODING'` error code is used when an invalid or unknown
+file encoding is passed.
```

Comment:
Sure, I think it would be better if this is:

```
The `'ERR_UNKNOWN_ENCODING'` error code is used when an invalid or unknown character encoding is passed to a method.
```